src/Stack/Upload.hs

```diff
@@ -126,13 +126,14 @@ applyCreds creds req0 = do
 -- sending a file like 'upload', this sends a lazy bytestring.
 --
 -- Since 0.1.2.1
-uploadBytes :: HackageCreds
+uploadBytes :: String -- ^ Hackage base URL
```

Comment:
Yes, it is, for the library but not the executable. Does it need to be documented somewhere?

Reply:
Perhaps in the changelog. I'll see what the other reviewer says.

I was pointing it out as it means this will be merged for the next major release (which is due very soon) — I'm just trying to maintain good habits.